lib/wasi/src/syscalls/wasix/bus_close.rs

```diff
@@ -0,0 +1,12 @@
+use super::*;
+use crate::syscalls::*;
+
+// FIXME: remove , since it's no longer used.
```

Comment:
So this need to be removed? Shall I do it?

Reply:
Already did so before seeing your comments.